demo-shell/src/app/components/cloud/community/community-task-cloud.component.ts

```diff
@@ -29,7 +29,7 @@ const TASK_FILTER_PROPERTY_KEYS = 'adf-edit-task-filter';
 
 @Component({
     templateUrl: './community-task-cloud.component.html',
-    styles: [`.adf-cloud-layout-tab-body .mat-tab-body-wrapper { height: 100%; }`]
+    styleUrls: ['./community-task-cloud.component.scss']
```

Comment:
needs ViewEncapsulation.none to allow deep theming

Reply:
done